content/ja/docs/tutorials/_index.md

```diff
@@ -29,7 +29,7 @@ content_template: templates/concept
 
 ## ステートレスアプリケーション
 
-* [クラスタ内のアプリケーションにアクセスするために外部IPアドレスを公開する](/docs/tutorials/stateless-application/expose-external-ip-address/)
+* [クラスタ内のアプリケーションにアクセスするために外部IPアドレスを公開する](/ja/docs/tutorials/stateless-application/expose-external-ip-address/)
```

Comment:
This should stay as `/docs/tutorials/stateless-application/expose-external-ip-address/` I think.

Reply:
Okay, I'll fix it.